pkg/contexts/ocm/transfer/transferhandler/standard/options.go

```diff
@@ -14,6 +14,10 @@ import (
 	"github.com/open-component-model/ocm/pkg/utils"
 )
 
+func init() {
+	transferhandler.RegisterHandler(1000, &TransferOptionsCreator{})
```

Comment:
How come this is suddenly needed here?

Reply:
This is the fallback transfer option set handler used in a second stage to map a set of given options to an appropriate
transfer handler. 

THis mechanism is only used for standard handlers used to operation flavors only taking transfer options and no transfer handler.  The the set of given options is used to determine the best matching handler implementation.
This simplifies the convenience operations, they just take options and the caller does not need to deal with transferhandler selection and conguration.

 If a custom handler is used, another operation flavor directly taking a transferhandler should be used.